### PR TITLE
net/ng_netapi: fixed implementation

### PR DESCRIPTION
**Pull request by haukepetersen**
- renamed opt variable in _get_set
- fixed msg type MACRO names

sometimes you wonder what slips through...

### CODE REVIEW

File: sys/net/crosslayer/ng_netapi/ng_netapi.c [hunk 1]
```diff
@@ -39,15 +39,15 @@ static inline int _get_set(kernel_pid_t pid, uint16_t type,
 {
     msg_t cmd;
     msg_t ack;
-    ng_netapi_opt_t opt;
+    ng_netapi_opt_t o;
     /* set ńetapi's option struct */
```
Comment (authmillenon) [inline sys/net/crosslayer/ng_netapi/ng_netapi.c]: Can you remove the accent above the 'n'  while you're at it?

**Comment by authmillenon** [comment]
Kicked Travis

**Comment by authmillenon** [comment]
Apart from my comment: ACK, when Travis is happy too.

**Comment by haukepetersen** [comment]
kicked Travis again...

**Pull request merged**